dmp/user-story-single-dmp-connections.ipynb

```diff
@@ -166,11 +166,6 @@
     "          }\n",
```

Comment:
<p>in line 150, Datum.award, i don’t think it exists in the query, so it will always return null. </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datacite/pidgraph-notebooks-python/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Ah yes, I missed that when moving my changes from within the binder site to the git repo. I added 'award: awardUri' to the query.